server/app.js

```diff
@@ -22,7 +24,11 @@ module.exports = (cfg) => {
   app.set('cfg', cfg);
   app.set('client:postmark', postmarkClient);
   app.set('client:psql', psqlPool);
-
+  app.use((req, res, next) => {
+    cfg.env === PRODUCTION // eslint-disable-line
```

Comment:
This doesn't make sense. If we're in the production environment, the middleware will:
1. not call `next()`
2. not return anything
3. not send a response

This breaks the server. Take another look at the `sslify` docs.

Reply:
> [In case of a non-encrypted HTTP request, express-sslify automatically redirects to an HTTPS address using a 301 permanent redirect.](https://github.com/florianheinemann/express-sslify)

From my understanding of this, anyone who tries to hit any route with a GET request will be redirected to the https address, so will make a new call to the server. I guess I will need to call next after `enforce.HTTPS()` is called, so this is allowed through, but I'm not sure what the need for point 2 & 3 are.

Reply:
Look at the examples. None of them look like that ☝️. You're just creating a middleware that does nothing.

Reply:
So...
```js
if (cfg.env === PRODUCTION) {
   app.use(enforce.HTTPS({ trustProtoHeader: true }));
};
```
Which is weird because my previous code was returning `403` for POST requests in tests, but maybe that was more to do with scope than design 🤷🏽‍♀️